test/Polly.Caching.Tests/HybridCacheResiliencePipelineBuilderExtensionsTests.cs

```diff
@@ -1,30 +1,54 @@
+using System.Text.Json;
 using Microsoft.Extensions.Caching.Hybrid;
 using Microsoft.Extensions.DependencyInjection;
 using Polly.Caching;
-using Polly.Testing;
 using Shouldly;
 
 namespace Polly.Caching.Tests;
 
 public class HybridCacheResiliencePipelineBuilderExtensionsTests
 {
     [Fact]
-    public void NonGeneric_AddHybridCache_BuildsStrategy()
+    public async Task NonGeneric_AddHybridCache_BuildsAndCaches()
     {
         var services = new ServiceCollection().AddHybridCache();
         using var provider = services.Services.BuildServiceProvider();
         var cache = provider.GetRequiredService<HybridCache>();
 
-        var options = new HybridCacheStrategyOptions { Cache = cache };
+        var options = new HybridCacheStrategyOptions { Cache = cache, CacheKeyGenerator = _ => "builder-key" };
```

Comment:
Style nit: if more than one property is set, set over multiple lines:
```suggestion
        var options = new HybridCacheStrategyOptions
        {
            Cache = cache,
            CacheKeyGenerator = _ => "builder-key",
        };
```